### PR TITLE
Pending status

### LINKED ISSUE #29: Status of Pending for Stops
Requested during PVTA meeting on April 30th, 2018:

Show that an incomplete stop is pending after someone started servicing it.

### PR DESCRIPTION
**Pull request by jforgue**
Closes #29

### CODE REVIEW

**Comment by dfaulken** [comment]
A few questions:

1. Why is it called arr if it's a hash?
2. Why are the keys symbols, if we're just going to have to translate them into strings in the view?

Good call on separating pending, completed, and not started. I think pending and not completed are the same thing conceptually, so we should be careful about distinguishing not started, started but not completed i.e. pending, and completed.

**Comment by dfaulken** [comment]
Also, what order do we think we should be putting these three groups in? Should it be according to priority, that is, pending, not started, then completed? Or should it feel more chronological, that is, not started, pending, then completed?

**Comment by jforgue** [comment]
I think ordering by priority would be more convenient for the user.

**Comment by jforgue** [comment]
Here's a screenshot
screen shot 2018-06-19 at 4 42 49 pm

File: app/views/bus_stops/by_route.haml [hunk 1]
```diff
@@ -3,14 +3,10 @@
...
   No stops on this route
 - else
-  - @stops.each_pair do |complete, stops|
-    - if complete
-      Completed
-    - else
-      Not Completed
-    Stops
+  - @stops_hash.each_pair do |status, stops|
+    = status.humanize + ' Stops'
```
Comment (dfaulken) [inline app/views/bus_stops/by_route.haml]: But again, why do this translation? Why not just put the string that we want into the view as the keys in the hash?

File: app/views/bus_stops/by_route.haml [hunk 2]
```diff
@@ -3,14 +3,10 @@
...
-      Not Completed
-    Stops
+  - @stops_hash.each_pair do |status, stops|
+    = status + ' Stops'
     %ul
     - stops.each do |stop|
       %li
         = link_to edit_bus_stop_path(stop.hastus_id) do
-          - stop.name
+          - stop.name
```
Comment (Anbranin) [inline app/views/bus_stops/by_route.haml]: Could you fix your text editor so it doesn't remove newlines from the end of files? I'm so sick of seeing this everywhere.

Comment (dfaulken) [reply app/views/bus_stops/by_route.haml]: I am a fan of reviewing with `?w=1` as the querystring.

Comment (werebus) [reply app/views/bus_stops/by_route.haml]: :+1: Or, there's actually an option in the "Diff settings" menu now.

<sub>Or, you know http://www.dealwithit.org/</sub>

**Pull request merged**